test_runner/src/main/kotlin/ftl/config/Device.kt

```diff
@@ -12,9 +12,9 @@ data class Device(
     val model: String,
     val version: String,
     val locale: String = defaultLocale,
-    val orientation: String = defaultOrientation
+    val orientation: String = defaultOrientation,
+    val isVirtual: Boolean? = null
```

Comment:
`isVirtual` shouldn't be nullable. We don't need nulls in later calculations.

Reply:
Alright! Changed! 👍